core/silkworm/chain/protocol_param.hpp

```diff
@@ -52,8 +52,10 @@ namespace fee {
 
 namespace param {
 
-    // https://eips.ethereum.org/EIPS/eip-170
-    inline constexpr size_t kMaxCodeSize{0x6000};
+    inline constexpr size_t kMaxCodeSize{0x6000};  // EIP-170
+
+    inline constexpr uint64_t kMinGasLimit{5000};
+    inline constexpr uint64_t kMaxGasLimit{INT64_MAX};  // EIP-4803
```

Comment:
Do we really need to re-define an already existent constant ? Can't we alias it with `using` ?